### PR TITLE
Implement deploy script for contracts stack

### PR DESCRIPTION
**Pull request by xPi2**
Task details: https://app.dework.xyz/nation3/nation3-court-v1-p?taskId=85c1f9c7-95f9-4eb5-878f-0f0db169b3fb

### CODE REVIEW

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@xPi2 This PR has 16 commits, but I think only one of these commits is the actual "Implement deploy script for contracts stack"?  Where are the other changes coming from?

**Comment by xPi2** [comment]
Sorry @aahna-ashina I created the draft before the merge of the collateral agreement branch from this branch forked, so it included all the changes from that branch also. Now is more clear what it's included in the PR

**Review by gregfromstl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ Run solidity tests:
 forge test
 ```
 
+### Deploy local
+
+Start Anvil node:
```
Comment (aahna-ashina) [inline README.md:30]: @xPi2 Include a link to Anvil installation instructions.

File: script/CollateralAgreementDeployer.s.sol [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
```
Comment (aahna-ashina) [inline script/CollateralAgreementDeployer.s.sol:1]: `GPL-3.0-or-later`

**Review by aahna-ashina — COMMENTED** [review]
👍

**Pull request merged**